### PR TITLE
Hide slideshow arrows when not hovering

### PR DESCRIPTION
**Pull request by ClementPasteau**

### CODE REVIEW

File: newIDE/app/src/UI/Slideshow/Slideshow.js [hunk 1]
```diff
@@ -98,34 +98,105 @@ const Slideshow = ({
     itemMobileRatio,
   });
   const classesForImage = useStylesForImage();
+  const containerRef = React.useRef(null);
+  const [isOverImage, setIsOverImage] = React.useState(false);
+  const outImageTimeoutId = React.useRef(null);
```
Comment (AlexandreSi) [inline newIDE/app/src/UI/Slideshow/Slideshow.js:103]: I would name this `leftImageRecentlyTimeoutId` instead

File: newIDE/app/src/UI/Slideshow/Slideshow.js [hunk 2]
```diff
@@ -98,34 +98,105 @@ const Slideshow = ({
...
+      // It's important to wait for the items, so that the listeners are
+      // created when the carousel is actually ready.
+      if (!containerElement || !items) return;
+
+      // Add event listeners on component mount. There is no need to
+      // remove them with a cleanup function because this element
+      // does not change and they will be destroyed when the element is
+      // removed from the DOM.
+      containerElement.addEventListener('mouseover', handleOverImage);
+      containerElement.addEventListener('mouseleave', handleOutImage);
```
Comment (AlexandreSi) [inline newIDE/app/src/UI/Slideshow/Slideshow.js:196]: Why don't you use onMouseEnter and onMouseLeave callbacks from react on the div?

Comment (4ian) [reply newIDE/app/src/UI/Slideshow/Slideshow.js:196]: btw! prefer pointer events: onPointerOver/onPointerLeave or pointerover pointerleave

Comment (ClementPasteau) [reply newIDE/app/src/UI/Slideshow/Slideshow.js:196]: Yeah forgot I could use the callbacks from react, I've switched that 👍

**Review by AlexandreSi — APPROVED** [review]

**Pull request merged**